### PR TITLE
Same token for studio & api.deepomatic.com

### PR DESCRIPTION
**Pull request by Alexis-Jacob**
With this PR you only need one token to interact with all the deepomatic's services. 

> Maybe it's better to tag deepomatic-client-python and then use the tag in the requirements ?

### CODE REVIEW

File: deepomatic/cli/cmds/feedback.py [hunk 1]
```diff
@@ -3,9 +3,15 @@
 import os
 import sys
 import logging
+import requests
 import threading
+import platform
 from tqdm import tqdm
-from .studio_helpers.http_helper import HTTPHelper
+from deepomatic.api.exceptions import (BadStatus, ServerError,
```
Comment (maingoh) [inline deepomatic/cli/cmds/feedback.py]: I think you can remove all those new imports

File: deepomatic/cli/cmds/feedback.py [hunk 2]
```diff
@@ -22,11 +28,12 @@
 
 
 class Client(object):
-    def __init__(self, token=None, verify_ssl=True, check_query_parameters=True, host=None, user_agent_suffix='', pool_maxsize=GREENLET_NUMBER):
+    def __init__(self, token=None, verify_ssl=True, check_query_parameters=True, host=None, user_agent_prefix='', pool_maxsize=GREENLET_NUMBER):
         if host is None:
             host = API_HOST
 
-        self.http_helper = HTTPHelper(token, verify_ssl, host, check_query_parameters, user_agent_suffix, pool_maxsize)
+        self.http_helper = HTTPHelper(api_key=token, verify_ssl=None, host=host, check_query_parameters=check_query_parameters,
```
Comment (maingoh) [inline deepomatic/cli/cmds/feedback.py]: put back `verify_ssl=verify_ssl` ?

File: deepomatic/cli/cmds/feedback.py [hunk 3]
```diff
@@ -39,11 +46,13 @@ def get_all_files_with_ext(path, supported_ext, recursive=True):
         for file in os.listdir(path):
             file_path = os.path.join(path, file)
             if recursive:
-                all_files.extend(get_all_files_with_ext(file_path, supported_ext))
+                all_files.extend(get_all_files_with_ext(
```
Comment (maingoh) [inline deepomatic/cli/cmds/feedback.py]: Your indent policy look weird to me ^^

File: deepomatic/cli/cmds/feedback.py [hunk 4]
```diff
@@ -22,11 +28,12 @@
...
 
 class Client(object):
-    def __init__(self, token=None, verify_ssl=True, check_query_parameters=True, host=None, user_agent_suffix='', pool_maxsize=GREENLET_NUMBER):
+    def __init__(self, token=None, verify_ssl=True, check_query_parameters=True, host=None, user_agent_prefix='', pool_maxsize=GREENLET_NUMBER):
         if host is None:
             host = API_HOST
 
-        self.http_helper = HTTPHelper(token, verify_ssl, host, check_query_parameters, user_agent_suffix, pool_maxsize)
+        self.http_helper = HTTPHelper(api_key=token, verify_ssl=None, host=host, check_query_parameters=check_query_parameters,
+                                      user_agent_prefix=user_agent_prefix, pool_maxsize=pool_maxsize, version=None)
```
Comment (leo-p) [inline deepomatic/cli/cmds/feedback.py]: Should this be `version=__version__`?

Comment (Alexis-Jacob) [reply deepomatic/cli/cmds/feedback.py]: Non that's the only trick of this PR, the HTTPHelper use this value to prefix the url with something with `/v{version}`, but in studio we don't have the concept of version.
Related to https://github.com/Deepomatic/deepomatic-client-python/pull/75

File: deepomatic/cli/cmds/feedback.py [hunk 5]
```diff
@@ -22,11 +22,12 @@
 
 
 class Client(object):
-    def __init__(self, token=None, verify_ssl=True, check_query_parameters=True, host=None, user_agent_suffix='', pool_maxsize=GREENLET_NUMBER):
+    def __init__(self, token=None, check_query_parameters=True, host=None, user_agent_prefix='', pool_maxsize=GREENLET_NUMBER):
```
Comment (maingoh) [inline deepomatic/cli/cmds/feedback.py]: I guess now you can do something like:
```
from ..version import __title__, __version__
DEFAULT_USER_AGENT_PREFIX = user_agent_prefix = '{}/{}'.format(__title__, __version__)

def __init__(self, **kwargs):
    kwargs['user_agent_prefix'] = kwargs.get('user_agent_prefix', DEFAULT_USER_AGENT_PREFIX)
    self.http_helper = HTTPHelper(version=None, **kwargs)
```

And later on:
```
client = Client(api_key=api_key)
```

Comment (maingoh) [reply deepomatic/cli/cmds/feedback.py]: Or just pass the `user_agent_prefix` at Client construction also

**Pull request merged**